src/features/buy-now/BuyNowModal.tsx

```diff
@@ -0,0 +1,23 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Types Imports
+import { BasicModalProps } from '../../lib/types/ui';
+
+//- Components Imports
+import { Modal } from '@zero-tech/zui/src/components';
+
+//- Container Imports
+import BuyNow from './BuyNow';
+
+export interface BuyNowModalProps extends BasicModalProps {}
+
+export const BuyNowModal: FC<BuyNowModalProps> = ({
+	open,
+	trigger,
+	onOpenChange,
+}) => (
+	<Modal onOpenChange={onOpenChange} trigger={trigger} open={open}>
```

Comment:
nit: can do something like

```
BuyNowModal = (props) => <Modal {...props} ...
```

Since `BuyNowModalProps` and `BasicModalProps` are the equal.

Reply:
Done